performance_regression_cdc_test.py

```diff
@@ -0,0 +1,269 @@
+        self.truncate_base_table(node1, cdclog_table=True)
+
+        node1.run_cqlsh(f"ALTER TABLE {self.keyspace}.{self.table} WITH cdc = {{'enabled': true, 'preimage': true}}")
+
+        _ = self._workload_cdc(write_cmd, 2,  # pylint: disable=unused-variable
+                               test_name="test_write",
+                               sub_type="cdc_preimage_enabled")
+
+        self.wait_no_compactions_running()
+
+        self.check_regression_with_baseline(base_line="cdc_disabled")
+
+    def test_mixed(self):  # pylint: disable=too-many-locals,unused-variable
+        self.keyspace = "keyspace1"
+        self.table = "standard1"
+        write_cmd = self.params.get("stress_cmd_w")
+        read_cdc_log_cmd = self.params.get("stress_cmd_r")
+
+        _ = self._workload_cdc(write_cmd, 2,  # pylint: disable=unused-variable
+                               test_name="test_mixed",
+                               sub_type="cdc_disabled")
+
+        node1: BaseNode = self.db_cluster.nodes[0]
+        self.wait_no_compactions_running()
+        self.run_fstrim_on_all_db_nodes()
+        self.truncate_base_table(node1)
+        node1.run_cqlsh(f"ALTER TABLE {self.keyspace}.{self.table} WITH cdc = {{'enabled': true}}")
+
+        _ = self._workload_cdc(write_cmd, 2,  # pylint: disable=unused-variable
+                               test_name="test_mixed",
+                               sub_type="cdc_enabled",
+                               cdc_read_cmd=read_cdc_log_cmd)
+
+        self.wait_no_compactions_running()
+        self.run_fstrim_on_all_db_nodes()
+        self.truncate_base_table(node1, cdclog_table=True)
+        node1.run_cqlsh(f"ALTER TABLE {self.keyspace}.{self.table} WITH cdc = {{'enabled': true, 'preimage': true}}")
+
+        _ = self._workload_cdc(write_cmd, 2,  # pylint: disable=unused-variable
+                               test_name="test_mixed_with_cdc_preimage",
+                               sub_type="cdc_preimage_enabled",
+                               cdc_read_cmd=read_cdc_log_cmd)
+        self.wait_no_compactions_running()
+
+        self.check_regression_with_baseline(base_line="cdc_disabled")
+
+    def _workload_cdc(self, stress_cmd, stress_num, test_name, sub_type=None, keyspace_num=1, prefix='', debug_message='',  # pylint: disable=too-many-arguments,too-many-locals
+                      save_stats=True, cdc_read_cmd=None):
+        read_statistics = None
+        future = None
+        if debug_message:
+            self.log.debug(debug_message)
+
+        if save_stats:
+            self.create_test_stats(sub_type=sub_type,
+                                   doc_id_with_timestamp=True,
+                                   append_sub_test_to_name=True)
+
+        stress_queue = self.run_stress_thread(stress_cmd=stress_cmd, stress_num=stress_num, keyspace_num=keyspace_num,
+                                              prefix=prefix, stats_aggregate_cmds=False)
+        if cdc_read_cmd:
```

Comment:
not sure I like the idea of this stress tool hidden here.
would rather see it in a separated file, so it can be reused again. if needed. 
and inside docker is possible, so we don't need to change loader images for things like that.

Reply:
will be moved to new PR